metricbeat/module/golang/expvar/_meta/data.json

```diff
@@ -0,0 +1,922 @@
+{
```

Comment:
This should be removed, as expvar can have any content. Otherwise people think it is only for betas.